src/Components/WeatherDetails/WeatherAdditionalDetails.js

```diff
@@ -0,0 +1,66 @@
+import React, { Component } from 'react';
+
+import WeatherAdditionalDetailsItem from './WeatherAdditionalDetailsItem';
+import { FadeInTop } from '../../common/Animations/Animations';
+
+class WeatherAdditionalDetails extends Component {
+  state = {
+    active: false,
+    data: null,
+  };
+
+  handleStateReachedData = data => {
+    this.setState({ data });
+  };
+
+  handleStateNotReachedData = () => {
+    this.setState({ data: null });
+  };
+
+  getAdditionalData = () => {
+    const { lat, lon } = this.props;
+    const API_ENDPOINT = `${process.env.REACT_APP_WEATHER_API_FORECAST_URL}lat=${lat}&lon=${lon}&cnt=6${process.env.REACT_APP_WEATHER_API_KEY}`;
+
+    fetch(API_ENDPOINT)
+      .then(response => response.json())
+      .then(json => this.handleStateReachedData(json))
```

Comment:
as this method is for getting data you can set the new state here when the data comes.

.then(json => this.setState({ data: json});
.catch(() => this.setState({error: true});

and render` <ErrorMessage > `component if the error is in the state. Inform user somehow that something was wrong while fetching the data